packages/wrangler/src/type-generation.ts

```diff
@@ -95,6 +95,12 @@ export async function typesHandler(
 		rules: config.rules,
 		queues: config.queues,
 		constellation: config.constellation,
+		send_email: config.send_email,
+		vectorize: config.vectorize,
+		hyperdrive: config.hyperdrive,
+		mtls_certificates: config.mtls_certificates,
+		browser: config.browser,
+		ai: config.ai,
```

Comment:
Can we change the type of this object to make sure we catch issues like this in the future? It should be something like `Omit<EnvironmentNonInheritable, "define" | "unsafe" | "tail_consumers" | "constellation" | "cloudchamber">`